op-node/cmd/batch_decoder/main.go

```diff
@@ -186,6 +188,48 @@ func main() {
 				return nil
 			},
 		},
+		{
+			Name:  "reassemble-devnet",
```

Comment:
We should not add another command that is specific for the devnet, we should add override config to the other command